reachc: error: RE0079

 ./Err_RecursionDepthLimit.rsh:4:23:application

 4|   const k = (x) => f(x(x));

    Recursion depth limit exceeded, more than 65536 calls; who would need more than that many?

Trace:
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  in "x" from (./Err_RecursionDepthLimit.rsh:4:17:function exp) at (./Err_RecursionDepthLimit.rsh:4:23:application)
  ...

For further explanation of this error, see: docs.reach.sh/ref-error-codes.html#%28part._.R.E0079%29

